### ISSUE TITLE
Install media not available by using USB 3.0 device

### ISSUE THREAD

**Issue by drscream** [omniosorg/illumos-omnios#505]
It's not possible to install OmniOS if the install media is an USB 3.0 flash drive because the device is not detected during boot. This issue is only related to USB 3.0 flash drives, so the installation works by using an USB 2.0 stick.

It might be related to:

- https://github.com/joyent/smartos-live/issues/693
- https://github.com/omniosorg/illumos-omnios/issues/121

**Comment by chincheta0815** [comment]
Just to add: I still encounter the same issue on OpenIndiana asof August, 11th latest update set.

**Comment by Laikulo** [comment]
I ran into this (possibly a recidivism) on r150105r using a USB3 ODE (which presents as a USB optical disk)

USB3+ODD is a really uncommon combination, as most 'real' optical drives can't reach above USB2 speeds, so often don't implement superspeeed.

I worked around the issue by using a USB2-only cable to downgrade the link.

I also found that the loader does not function when the ODE is configured as a multi-LUN device, though this is a pretty common issue with devices like this, so it's standard practice to put the ODE in single-lun mode when this happens.
